Fixes this error:

error[E0433]: failed to resolve: use of undeclared type `Pid`
   --> src/exec.rs:753:15
    |
753 |     p.set_pid(Pid::new(pid).unwrap());
    |               ^^^ use of undeclared type `Pid`
    |
help: consider importing this struct
    |
6   + use crate::proc::Pid;
    |

after: https://github.com/fish-shell/fish-shell/commit/eb4cec1fe25336d07aeed6f67f4bcd0f0cb2f8f0

--- a/src/exec.rs
+++ b/src/exec.rs
@@ -33,7 +33,6 @@ use crate::libc::_PATH_BSHELL;
 use crate::nix::{getpid, isatty};
 use crate::null_terminated_array::OwningNullTerminatedArray;
 use crate::parser::{Block, BlockId, BlockType, EvalRes, Parser};
-#[cfg(FISH_USE_POSIX_SPAWN)]
 use crate::proc::Pid;
 use crate::proc::{
     hup_jobs, is_interactive_session, jobs_requiring_warning_on_exit, no_exec,
